testsuite/MDAnalysisTests/transformations/test_nojump.py

```diff
@@ -0,0 +1,49 @@
+###
+
+import numpy as np
+import pytest
+from numpy.testing import assert_array_almost_equal
+
+import MDAnalysis as md
+from MDAnalysis.transformations import NoJump
+from MDAnalysisTests import datafiles as data
+
+@pytest.fixture()
+def nojump_universes():
+    """
+    Create the universe objects for the tests.
+    """
+    u = md.Universe(data.PSF_TRICLINIC, data.DCD_TRICLINIC)
+    transformation = NoJump()
+    u.trajectory.add_transformations(transformation)
+    return u
+
+@pytest.fixture()
+def nojump_universes_nocheck():
+    """
+    Create the universe objects for the tests.
+    Do not check for continunity
+    """
+    u = md.Universe(data.PSF_TRICLINIC, data.DCD_TRICLINIC)
+    transformation = NoJump(check_continuity=False)
+    u.trajectory.add_transformations(transformation)
+    return u
+
+def test_nojump_fwd(nojump_universes):
+    """
+    Test if the nojump transform is returning the correct
+    values when iterating forwards over the sample trajectory.
+    """
+    ref_matrix_fwd1 = np.asarray([-3.42261, 11.28495, -1.37211])
```

Comment:
What are the red values and how did you obtain them? Add a comment for provenance.

Reply:
My workflow for testing this was probably non-ideal. I basically made water orthogonal and non-orthogonal water boxes, checked that the results were equivalent to what I got from VMD/fastpbc. I then ran the algorithm on existing triclinic systems, and did some sanity checks. These are two atomic positions that happened to move the most. Comments are now included to highlight this.

Reply:
@jvermaas  May I suggest making a test case with a single particle that travels at a constant speed through the box? That way it should be easy to verify the correcness of the unwrapping. You can do this directly  in MDAnalysis using

```python
import MDAnalysis 
import numpy as np

N = 1
u = mda.Universe.empty(N, trajectory=True)
coordinates = np.empty((1000,  # number of frames
                        u.atoms.n_atoms,
                        3))
coordinates = ... # generate wrapped coordinates for a particle moving at a constant speed in the box. 
# I can help you with this if you need. 
u.load_new(coordinates, order='fac')
```
You can then make this universe a test fixture.

Reply:
Oooh... That is cleverer @hmacdope . I'll let you know how I make out. I'll probably need to look at how to add PBC information to a universe, but there is probably documentation somewhere.

Reply:
I am going to make this comment a blocking review @jvermaas, let me know if you need any help, feel free to just ping me.

Reply:
I really like @hmacdope 's idea. 

$$
\mathbf{r}(t) = \mathbf{v}_0 t + \mathbf{r}_0
$$

with some odd angle of the initial velocity and a smallish box and then wrap the straight shot trajectory with the  `wrap()` transform.

Reply:
RE setting PBC the following should work to give you a 10x10x10 orthorhombic box using the `dimensions` argument. 

import MDAnalysis 
import numpy as np

N = 1
u = mda.Universe.empty(N, trajectory=True)
coordinates = np.empty((1000,  # number of frames
                        u.atoms.n_atoms,
                        3))

u.load_new(coordinates, order='fac', in_memory=True, dimensions=[10,10,10, 90.0, 90.0,90.0])
```

Reply:
I scrapped the old benchmark and replaced it with a synthetic one. I think its still intelligible, and I can explain why the non-orthogonal number is what it is.